.travis.yml

```diff
@@ -7,6 +7,7 @@ node_js:
 
 before_script:
   - npm install
+  - npm run prettier:ci
```

Comment:
This should probably be in `script` rather than `before_script`.  In general, I think of `before_script` as anything that shouldn't reasonably fail, while `script` are things that one might expect to fail if someone made a mistake while working on the project.

Reply:
Moved it to `script`